lib/src/controls/yaru_popup_menu_button.dart

```diff
@@ -40,10 +40,20 @@ class YaruPopupMenuButton<T> extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ClipRRect(
-      borderRadius: BorderRadius.circular(kYaruButtonRadius),
+    final style = OutlinedButtonTheme.of(context).style;
+    final state = <MaterialState>{if (!enabled) MaterialState.disabled};
+    final side = style?.side?.resolve(state);
+    final shape = style?.shape?.resolve(state) ??
+        RoundedRectangleBorder(
+          side: BorderSide(color: Theme.of(context).dividerColor),
+          borderRadius: BorderRadius.circular(kYaruButtonRadius),
+        );
+    return DecoratedBox(
```

Comment:
the order of DecoratedBox + clipping Material is important to avoid clipping parts of the border when it lands on subpixels and gets antialiased